docs/en/operation/Metrics-System.md

```diff
@@ -71,6 +76,9 @@ If Alluxio is deployed in a cluster, this file needs to be distributed to all th
 Then, start Alluxio, CSV files containing metrics will be found in the `sink.csv.directory`. The
 file name will correspond with the metric name.
 
+All possible sink specific configurations can be found within their respective class files: 
+`$ALLUXIO_HOME/core/common/src/main/java/alluxio/metrics/sink/`. 
```

Comment:
Same comment as above about users having access to this directory.

Also, it would probably be pretty hard to users to figure out the configuration just by looking at the source file (it's unfortunately not very straightforward). I would probably just tell them to look at the `metrics.properties.template`

Reply:
ah i see, thanks for clarifying! updated again